[DOCS] Adds content on ad-hoc data views (#146250)
elastic/kibana: docs/concepts/data-views.asciidoc

> === Create a {data-source}

Reviewer: ```suggestion
=== Create {data-sources}
```

you get a {data-source} for free, and can start exploring your data.
If you loaded your own data, follow these steps to create a {data-source}.

> . Open *Lens* or *Discover*, and then open the data view menu.

Reviewer: Can we add links here for Lens and Discover?

+
[role="screenshot"]
image:management/index-patterns/images/create-data-view.png["Create data view"]
+
** To match multiple sources, use a wildcard (*). `filebeat-*` matches
`filebeat-apache-a`, `filebeat-apache-b`, and so on.
+
** To match multiple single sources, enter their names,
separated by a comma.  Do not include a space after the comma.
`filebeat-a,filebeat-b` matches two indices.
+
** To exclude a source, use a minus sign (-), for example, `-test3`.

. Open the *Timestamp field* dropdown,
and then select the default field for filtering your data by time.
+
** If you don’t set a default time field, you can't use
global time filters on your dashboards. This is useful if
you have multiple time fields and want to create dashboards that combine visualizations
based on different timestamps.z
+
** If your index doesn’t have time-based data, choose *I don’t want to use the time filter*.

. Click *Show advanced settings* to:
** Display hidden and system indices.
** Specify your own {data-source} name. For example, enter your {es} index alias name.

. [[reload-fields]] Click *Save {data-source} to {kib}*.
+
You can manage your data view from *Stack Management*.

[float]

> ==== Temporary {data-sources}

Reviewer: ```suggestion
==== Create temporary {data-sources}
```

> What if you want to explore your data or create a visualization without first saving it as a data view?

Reviewer: ```suggestion
Want to explore your data or create a visualization without saving it as a data view?
```

> You can do that by selecting *Use without saving* in the *Create {data-source}* form in *Discover*

Reviewer: ```suggestion
Select *Use without saving* in the *Create {data-source}* form in *Discover*
```

or *Lens*. With a temporary {data-source}, you can
do everything you'd do with a saved

> {data-source},

Reviewer 2: well, in Discover you can't use "Share", this will convert the the temporary into a regular data view.

> including adding fields and creating an {es} query alert. Your work won't be visible to others in your space.

Reviewer: ```suggestion
such as adding fields and creating {es} query alerts. Your work won't be visible to others in your space.
```

A temporary {data-source} remains in your space until you change apps, or until you save it.

> Temporary {data-sources} are not available in *Stack Management.*

Reviewer: ```suggestion
The temporary {data-sources} you create are unavailable in *Stack Management.*
```

[role="screenshot"]
image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blte3a4f3994c44c0cc/637eb0c95834861044c21a25/ad-hoc-data-view.gif[how to create an ad-hoc data view]

[float]
[[rollup-data-view]]

> ==== Data views and rolled up data

Reviewer: Can this become actionable?

For example, `Use data views with rolled up data`

[float]
[[management-cross-cluster-search]]

> ==== Data views and {ccs}

Reviewer: Same here. What about:

`Use data views with cross-cluster search`
